.travis.yml

```diff
@@ -7,6 +7,5 @@ node_js:
   - '0'
   - '0.10'
 after_script:
-  - npm install istanbul-coveralls@1
```

Comment:
This line cannot be removed because `istanbul-coveralls` is used at [`.travis.yml#L11`](https://github.com/image-size/image-size/blob/32a1a54dc0b1a508296ef1abe17e413a3bf55b8b/.travis.yml#L11).